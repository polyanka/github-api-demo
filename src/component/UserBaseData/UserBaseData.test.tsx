import { shallow } from 'enzyme';
import * as React from 'react';

import { UserBaseData } from './index';

describe('<UserBaseData /> ', () => {
  it('renders correctly with props', () => {
    const user = { id: 1, login: 'login', avatar_url: 'avatar_url' };
    const wrapper = shallow(<UserBaseData user={user} />);
    expect(wrapper).toMatchSnapshot();
  });
});
