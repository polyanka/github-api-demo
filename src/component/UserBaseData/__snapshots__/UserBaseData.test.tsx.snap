// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserBaseData />  renders correctly with props 1`] = `
<WithStyles(ForwardRef(Card))
  style={
    Object {
      "maxWidth": 345,
      "padding": 16,
    }
  }
>
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(CardMedia))
      image="avatar_url"
      style={
        Object {
          "height": 140,
        }
      }
      title="login"
    />
    <WithStyles(ForwardRef(Typography))
      component="h2"
      gutterBottom={true}
      variant="h5"
    >
      login
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(CardContent))>
  <WithStyles(ForwardRef(CardActions))>
    <WithStyles(ForwardRef(Button))
      color="primary"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      size="medium"
      to="/user/login"
      variant="contained"
    >
      Open
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(CardActions))>
</WithStyles(ForwardRef(Card))>
`;
